### PR TITLE
test(solana): priority fees support [LIVE-11792]

### PR DESCRIPTION
**Pull request by Justkant**
# Checklist

- [x] App update process has been followed 
- [x] Target branch is `develop` 
- [ ] Application version has been bumped (no bump as we only update tests)

Related app-solana PR: https://github.com/LedgerHQ/app-solana/pull/97

I had to update the `solana_cmd_builder` to support more instructions
I also had to generate a golden run for the new tests

### CODE REVIEW

File: .github/workflows/reusable_swap_functional_tests.yml [hunk 1]
```diff
@@ -68,7 +68,7 @@ on:
 
       branch_for_solana:
         required: false
-        default: 'develop'
+        default: 'fix/LIVE-11792'
```
Comment (Justkant) [inline .github/workflows/reusable_swap_functional_tests.yml:71]: Added this in a separate commit that I can remove when we merge the fix on app-solana

File: ledger_app.toml [hunk 2]
```diff
@@ -6,6 +6,7 @@ devices = ["nanos", "nanox", "nanos+", "stax", "flex"]
 [use_cases]
 use_test_keys = "TEST_PUBLIC_KEY=1"
 dbg_use_test_keys = "DEBUG=1 TEST_PUBLIC_KEY=1"
+testing_dbg_use_test_keys = "DEBUG=1 TESTING=1 TEST_PUBLIC_KEY=1"
```
Comment (tdejoigny-ledger) [inline ledger_app.toml:9]: not sure to understand the need of this added line

Comment (Justkant) [reply ledger_app.toml:9]: It was useful for some swap e2e tests done locally with LL but I can remove it

File: .github/workflows/reusable_swap_functional_tests.yml (hunk 1 above)
Comment (tdejoigny-ledger) [reply .github/workflows/reusable_swap_functional_tests.yml:71]: 👍

**Review by yogh333 — APPROVED** [review]

**Review by tdejoigny-ledger — APPROVED** [review]

**Pull request merged**